augur/__init__.py

```diff
@@ -14,6 +14,7 @@
 from .errors import AugurError
 from .io import print_err
 from .utils import first_line
+from .argparse_ import add_default_command
```

Comment:
I named the module `argparse_` (with an underscore) to avoid overwriting the `argparse` standard library here.
I did this to fix the problem without really knowing why it was overwriting.

As @tsibley said in [Slack](https://bedfordlab.slack.com/archives/C032AT1NT/p1657671573248189), he found the [relevant docs](https://docs.python.org/3/reference/import.html#submodules) to explain this behavior: 
> It does seem like `from .x import y` makes a local variable `x` in scope

> Crucially, this only happens _when inside the package module_ (i.e. the `__init__.py` adjacent to `x.py`). If you're inside a sibling module in the same package, the same import statement _doesn't_ bind `x`.